tests/integration/acp_setup_js.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2025 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tests
+
+import (
+	"github.com/sourcenetwork/defradb/node"
+
+	"github.com/sourcenetwork/immutable"
+)
+
+func setupSourceHub(s *state) ([]node.DocumentACPOpt, error) {
+	return s.acpOptions, nil
```

Comment:
todo: I don't know if I missed this in my recent PR or this was result of a conflict that was resolved, can you please rename `s.acpOptions` to `s.documentACPOptions`.

Reply:
done